binaries/upload.py

```diff
@@ -94,15 +81,33 @@ def upload_conda_packages():
         required=False,
         help="Specify whether to upload to test PyPI",
     )
+
+    parser.add_argument(
+        "--dry_run",
+        action="store_true",
+        help="dry_run will print the commands that will be run without running them. Only works for pypi now",
```

Comment:
Why can't we use dry_run with conda? Is there some limitation with the conda command?

Reply:
There's a file traversal that messes things up, I think I can fix with mock files but lemme think more

Reply:
So after messing around a bit with mocking I think changing behavior here will make the dry_run command more confusing and create a false sense of security if things are changed, so will opt out of doing this for now